Sources/AsyncHTTPClient/AsyncAwait/HTTPClientResponse.swift

```diff
@@ -132,10 +187,10 @@ extension HTTPClientResponse.Body.Storage.AsyncIterator: AsyncIteratorProtocol {
 @available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
 extension HTTPClientResponse.Body {
     init(_ body: TransactionBody) {
-        self.init(.transaction(body))
+        self.init(.transaction(body), expectedContentLength: body.expectedContentLength)
     }
 
-    @usableFromInline init(_ storage: Storage) {
+    @usableFromInline init(_ storage: Storage, expectedContentLength: Int?) {
```

Comment:
```suggestion
    @usableFromInline init(_ storage: Storage) {
```
`expectedContentLength` is no longer used and can be removed.